### ISSUE TITLE
search box does not verk

### ISSUE THREAD

**Issue by 1ma** [jorge07/docker-packagist#1]
Try for instance http://localhost/search/?q=das, the app returns an HTTP 500 error.

For what it's worth, here's the stack trace:

```
[2016-10-10 22:06:28] request.INFO: Matched route "search.ajax". {"route_parameters":{"_controller":"Packagist\\WebBundle\\Controller\\WebController::searchAction","_route":"search.ajax"},"request_uri":"http://localhost/search/?q=das"} []

[2016-10-10 22:06:28] security.INFO: Populated the TokenStorage with an anonymous Token. [] []

[2016-10-10 22:06:28] request.CRITICAL: Uncaught PHP Exception Solarium_Client_HttpException: "Solr HTTP error: HTTP request failed" at /app/vendor/solarium/solarium/library/Solarium/Client/Adapter/Http.php line 80 {"exception":"[object] (Solarium_Client_HttpException(code: 0): Solr HTTP error: HTTP request failed at /app/vendor/solarium/solarium/library/Solarium/Client/Adapter/Http.php:80)\n[stacktrace]\n#0 /app/vendor/solarium/solarium/library/Solarium/Client/Adapter/Http.php(62): Solarium_Client_Adapter_Http->check(false, Array)\n#1 /app/vendor/solarium/solarium/library/Solarium/Client.php(574): Solarium_Client_Adapter_Http->execute(Object(Solarium_Client_Request))\n#2 /app/vendor/solarium/solarium/library/Solarium/Client.php(554): Solarium_Client->executeRequest(Object(Solarium_Client_Request))\n#3 /app/vendor/solarium/solarium/library/Solarium/Client.php(651): Solarium_Client->execute(Object(Solarium_Query_Select))\n#4 /app/vendor/pagerfanta/pagerfanta/src/Pagerfanta/Adapter/SolariumAdapter.php(153): Solarium_Client->select(Object(Solarium_Query_Select), NULL)\n#5 /app/vendor/pagerfanta/pagerfanta/src/Pagerfanta/Adapter/SolariumAdapter.php(122): Pagerfanta\\Adapter\\SolariumAdapter->createResultSet()\n#6 /app/vendor/pagerfanta/pagerfanta/src/Pagerfanta/Adapter/SolariumAdapter.php(102): Pagerfanta\\Adapter\\SolariumAdapter->getResultSet(0, 15)\n#7 /app/vendor/pagerfanta/pagerfanta/src/Pagerfanta/Pagerfanta.php(319): Pagerfanta\\Adapter\\SolariumAdapter->getSlice(0, 15)\n#8 /app/vendor/pagerfanta/pagerfanta/src/Pagerfanta/Pagerfanta.php(303): Pagerfanta\\Pagerfanta->getCurrentPageResultsFromAdapter()\n#9 /app/vendor/pagerfanta/pagerfanta/src/Pagerfanta/Pagerfanta.php(475): Pagerfanta\\Pagerfanta->getCurrentPageResults()\n#10 /app/src/Packagist/WebBundle/Controller/WebController.php(146): Pagerfanta\\Pagerfanta->getIterator()\n#11 [internal function]: Packagist\\WebBundle\\Controller\\WebController->searchAction(Object(Symfony\\Component\\HttpFoundation\\Request))\n#12 /app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php(144): call_user_func_array(Array, Array)\n#13 /app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php(64): Symfony\\Component\\HttpKernel\\HttpKernel->handleRaw(Object(Symfony\\Component\\HttpFoundation\\Request), 1)\n#14 /app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/ContainerAwareHttpKernel.php(69): Symfony\\Component\\HttpKernel\\HttpKernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)\n#15 /app/app/bootstrap.php.cache(1501): Symfony\\Component\\HttpKernel\\DependencyInjection\\ContainerAwareHttpKernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)\n#16 /app/web/app.php(14): Symfony\\Component\\HttpKernel\\Kernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request))\n#17 {main}"} []
```

**Comment by jorge07** [comment]
Follow the official page for a correct installation

**Comment by jorge07** [comment]
must be fixed with: e6a16ad41f9a35f3cebeff9b23dc89914d6ec934

**Comment by 1ma** [comment]
confirmed :+1: good job
